osd_cluster_create/byo-vpc.adoc

```diff
@@ -0,0 +1,34 @@
+[id="byo-vpc"]
+= Managing a Bring-Your-Own Virtual Private Cloud (VPC)
```

Comment:
Sentence case, so at least "bring-your-own" should be lower. Not sure if Virtual Private Cloud is always capitalized or not